js/Story.js

```diff
@@ -408,17 +408,18 @@ Story.prototype.blurTiddler = function(title)
 //#  tag - value of field, without any [[brackets]]
 //#  mode - +1 to add the tag, -1 to remove it, 0 to toggle it
 //#  field - name of field (eg "tags")
+//# 2023-10-23 Okido: this is not the correct variable, it must be tags
```

Comment:
no need for an additional comment in the code, this just fixes refactoring from ac8eb03 (in fact, I had a todo to test this refactoring more thoroughly, so thanks for this)